src/bin/ch-remote.rs

```diff
@@ -309,6 +314,20 @@ fn send_migration_api_command(
     .map_err(Error::ApiClient)
 }
 
+fn create_api_command(socket: &mut UnixStream, path: &str) -> Result<(), Error> {
+    let mut data = String::default();
+    if path == "-" {
+        std::io::stdin()
+            .read_to_string(&mut data)
+            .map_err(Error::ReadingStdin)?;
+    } else {
+        let mut f = std::fs::File::open(path).map_err(Error::ReadingFile)?;
+        f.read_to_string(&mut data).map_err(Error::ReadingFile)?;
```

Comment:
Using `std::fs::read_to_string(path)` would save one line.

Reply:
Dang, I thought there was a function like that and I couldn't find it. As this got automerged. I'll address this in a follow up commit.